salt/auth/mysql.py

```diff
@@ -55,10 +55,27 @@
 log = logging.getLogger(__name__)
 
 try:
+    log.debug('Trying to import MySQLdb')
```

Comment:
@cachedout Should we be logging in the namespace like this? Auth modules don't have a `__virtual__` method, but will these debug lines _always_ should up in logs?

Reply:
Yes they do.

https://github.com/saltstack/salt/blob/2017.7/salt/auth/pam.py#L155

Reply:
Ah, nice. I only checked this auth file and didn't see one. However, @gtmanfred rightly pointed out that anything that is loaded by the LazyLoader should/can have a `__virtual__()`.

@MaximilianMeister Perhaps this module should have one as well and then only log the type of connector once it has been detected. We don't want those log messages flooding people's logs if they're not using these files.

Can you update these files so that the log lines are inside a `__virtual__` function and gated by the appropriate checks? It can be something simple like:

```
def __virtual__():
    '''
    <whatever doc>
    '''
    if HAS_MYSQL_DB:
        log.debug('MySQLdb loaded')
        return True
    elif HAS_PYMYSQL:
        log.debug('PyMySQL loaded')
        return True
    return False, 'Either the \'MySQLdb\' or \'PyMySQL\' libraries must be installed.
```

You can add whatever other log lines you want there, but nothing can be logged outside of a check (`__virtual__` + `if` statement) so we don't flood the logs. Each file in this PR will need the same type of checks applied.